packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1512,19 +1513,51 @@ module.exports = (
 
         const { uid, scope } = request.auth.credentials;
         const { ecosystemAnonId } = request.payload;
-        const noneMatchHeader = request.headers['If-None-Match'];
         const scopeSet = ScopeSet.fromArray(scope);
 
+        function parseMultiValueHeader(named) {
+          const value = request.headers[named];
+          if (!value || !value.length) {
+            return [];
+          }
+          return value.toString().split(/,\s?/);
+        }
+
+        function hashAnonId(anonId) {
+          const hash = crypto.createHash('md5');
```

Comment:
I'd like us to standardize on `sha-256` for hashes. For a few reasons, but the one on the top of my head is that we can do native sha in the browser